bugbot/rules/uplift_beta.py

```diff
@@ -46,13 +46,19 @@ def handle_bug(self, bug, data):
         else:
             nickname = bug["assigned_to_detail"]["nick"]
 
+        if self.is_needinfo_on_assignee(bug.get("flags", []), assignee):
+            return None
+
         data[bugid] = {
             "id": bugid,
             "mail": assignee,
             "nickname": nickname,
             "summary": self.get_summary(bug),
             "regressions": bug["regressions"],
+            "assigned_to": assignee,
+            "flags": bug.get("flags", []),
         }
```

Comment:
Do we still need this?

Reply:
No, good catch. Removed here: 95106d1164132fa2bd317ebf2355e3c2de61e8b2